src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -14552,7 +14552,10 @@ namespace EnergyPlus::OutputReportTabular {
         // PURPOSE OF THIS SUBROUTINE:
         //   Convert to XML safe escaped character string
         //   so it excludes:
-        //               " ' < > &
+        //               " ' < > & degree-sign
+        //   If isXML=false, it does an HTML conversion, so only ` < > & ` and degree sign
+        //   Technically HTML4 doesn't support &quot, though most browsers would anyways.
+        //   Also, escaping single and double quotes is only needed inside attributes
```

Comment:
Mod `ConvertToEscaped` to add a `bool isXML` so it can be used for HTML escaping too, which is quite similar